### ISSUE TITLE
Preserved Terrarium directives are impossible

### ISSUE THREAD

**Issue by Azlier** [Skyrat-SS13/Skyrat-TG#23075]
Reporting client version: 514.1589

## Round ID:
9132

## Testmerges:
- [[MODULAR] Adds the option for missing a limb to the prosthetics menu](https://github.com/Skyrat-SS13/Skyrat-tg/pull/22097)
- [NRI/CIN Precursor Food Replicator](https://github.com/Skyrat-SS13/Skyrat-tg/pull/22679)

## Reproduction:

Play as a preserved terrarium. Attempt to cultivate the wastes. Immediately die because your benevolent masters failed to plan for your being completely unable to breathe the atmosphere in the place of your assignment. Your base has no masks and no tanks. Not even a radio to call for help in getting some. Salbutamol can't protect you from carbon dioxide. Suffer forever.

In other words, primal podpeople need to either be able to breathe in their work environments, have internals supplied to them, or have their objectives adjusted to not require they go out and die fruitlessly.

**Comment by PawnAtWork** [comment]
Do they not get kudzu or tobacco?

**Comment by Azlier** [comment]
They don't start with kudzu. Space tobacco salbutamol can compensate for lack of oxygen, but it can't protect you from high levels of CO2 causing unconsciousness and rapid death.
